cmd/cluster/create_cluster.go

```diff
@@ -212,4 +239,5 @@ func init() {
 	createClusterCmd.MarkFlagRequired("region-info")
 	createClusterCmd.Flags().String("preferred-region", "", "[OPTIONAL] The preferred region in a multi region cluster. The preferred region handles all read and write requests from clients.")
 	createClusterCmd.Flags().String("default-region", "", "[OPTIONAL] The primary region in a partition-by-region cluster. The primary region is where all the tables not created in a tablespace reside.")
+	createClusterCmd.Flags().Bool("enable-connection-pooling", false, "[OPTIONAL] Enable connection pooling for the cluster after creation. Default false. (Requires CONNECTION_POOLING feature flag)")
```

Comment:
I don't think we need to inform the customers about the feature flag. We need to remove it/make it true on our side when it is ready for the public.